### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Henkitonki** [WinMerge/winmerge#1476] [Q&A]
I compare 2  image folders. when I open either on left or right with the assigned application it opens with paint instead of photo!!
In my parameters "application by file types" for jpeg it is photos!

**Comment by sdottaka** [comment]
Please see the issue below.

https://github.com/WinMerge/winmerge/issues/321

If you want to open an image file with the Photos app, please use the Left or Right Shell Menu as follows

![image](https://user-images.githubusercontent.com/98126/188664048-b1b5bb5c-7895-4db2-9952-74349237105b.png)
